tests/Sourcing/EventSteamTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Krixon\DomainEvent\Test\Sourcing;
+
+use InvalidArgumentException;
+use Krixon\DomainEvent\Event;
+use Krixon\DomainEvent\Sourcing\EventStream;
+use Krixon\DomainEvent\Sourcing\EventStreamId;
+use PHPUnit\Framework\TestCase;
+
+class EventSteamTest extends TestCase
```

Comment:
Typo in the class name.

```suggestion
class EventStreamTest extends TestCase
```

Reply:
Fixed